src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixExtension.groovy

```diff
@@ -45,15 +45,24 @@ class ScalafixExtension {
 
     ScalafixExtension(Project project) {
         this.project = project
-        configFile = project.objects.fileProperty().convention(
-                locateConfigFile(project) ?: locateConfigFile(project.rootProject))
+        final boolean isGradle4 = project.gradle.gradleVersion.startsWith("4")
```

Comment:
sug.: we could introduce a new class `Gradle` with a `static boolean isVersion4(project)` method